org.eclipse.wildwebdeveloper.tests/.project

```diff
@@ -20,8 +20,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
This commit was accidental. 
I can do a git cherry pick when this patch is OK for a merge to get rid of this useless commit.